### DISCUSSION TITLE
RaspberryMatic-3.67.10.20230128-06ac2c-rpi3 - WebUI

### DISCUSSION THREAD

**Discussion by ghost** [OpenCCU/OpenCCU#2190] [🇩🇪 Fragen&Antworten] [answered]
Mir ist eine leichte Abweichung aufgefallen - es war vorher nicht da
chrome und firefox, rpi3
![1](https://user-images.githubusercontent.com/81189856/215271480-01ef10a7-cc8a-4c69-b9c7-41046ca704ac.jpg)

**Comment by jens-maus** [comment]
Und wie sah es vorher aus?

**Reply by ghost** [reply]
![2](https://user-images.githubusercontent.com/81189856/215271808-9a14f88c-8691-49dd-a211-e0d533a4903f.jpg)

**Reply by jens-maus** [reply]
Und seit wann ist das?

**Reply by ghost** [reply]
es war ok noch im Nightly Build : RaspberryMatic-3.67.10.20230124-fd96e4-rpi3

PS. gilt für HmIP-FBL

**Comment by ghost** [comment]
RaspberryMatic-3.67.10.20230124-fd96e4-rpi3 ------ sorry, hier war es auch nicht richtig
![2](https://user-images.githubusercontent.com/81189856/215275063-6a9e9851-cfea-4b53-b930-de9be180da2f.jpg)

**Comment by ghost** [comment]
es war gut hier - RaspberryMatic-3.67.10.20230114-rpi3 


![2](https://user-images.githubusercontent.com/81189856/215275713-e1e78df2-1851-472f-94d7-5d97fcfd3a86.jpg)

**Comment by ghost** [comment]
auf dem telefon ist webui praktisch unbrauchbar

**Reply by jens-maus** [reply]
Dafür ist es ja auch nicht gemacht und sollte auch nicht darauf verwendet werden.

**Reply by MichaelN0815** [reply]
Ich bitte dich. Das ging bisher ganz brauchbar und ist nicht das bootstrap Framework dafür gedacht Geräte unabhängig zu funktionieren?

**Reply by ghost** [reply]
aber es kann leicht gelöst werden - Sie müssen die Option "Desktop-Version" aktivieren

**Reply by jens-maus** [reply]
Und das wäre wie? Im Browser? Und eigentlich sollte bootstrap keinen effekt haben weil es ja noch nicht großflächig in der webui verwendet wird. Noch haben wir ja nicht angefangen web element gehen bootstrap element zu ersetzen.

**Comment by ghost** [comment]
vor :
![1](https://user-images.githubusercontent.com/81189856/215316432-8c0019cb-7135-4c34-a8bb-7ba6041223e1.jpeg)
Desktop Version -wir schalten es ein:
![2](https://user-images.githubusercontent.com/81189856/215316491-e26b2a00-b1ff-46c7-a270-25cde908dabd.jpeg)
nacher: 
![3](https://user-images.githubusercontent.com/81189856/215316494-e6a3bb7b-1d32-41c8-8bec-4aef1ec8c56c.jpeg)

**Reply by ghost** [reply]
vorher war kein unterschied
mobile oder Desktop-Version

**Reply by jens-maus** [reply]
Verstehe. Muss ich mal testen. Allerdings hat das mit dem ursprünglichen Thema hier (HmIP-FBL) nichts zu tun, oder? Weil dann gehört das separat gemeldet.

**Reply by ghost** [reply]
ja, das hat nichts mit HmIP-FBl zu tun

**Reply by jens-maus** [reply]
https://github.com/jens-maus/RaspberryMatic/commit/298f8aa4f25a406faadb57459013a73dce1ca94d sollte das Problem mit der Ansicht auf Mobilgeräten hoffentlich etwas verbessern.

**Comment by jens-maus** [comment, marked as answer]
Ok. Das Problem mit den komisch aussehenden Slidern und den zu hohen Eingabefeldern sollte mit dem nächsten nightly snapshot beseitigt sein. Siehe https://github.com/jens-maus/RaspberryMatic/commit/00b0c936a3221af33e5cf851b756345863f7d380
